plone/app/contenttypes/tests/test_webdav.py

```diff
@@ -0,0 +1,56 @@
+import os.path
+from plone.app.contenttypes.testing import (
+    PLONE_APP_CONTENTTYPES_INTEGRATION_TESTING,
+)
```

Comment:
Imports should be one liners, please. Use ``  # noqa`` to bypass flake8 checking.
```
from plone.app.contenttypes.testing import PLONE_APP_CONTENTTYPES_INTEGRATION_TESTING  # noqa
```